src/Internal/Wallet/Cip30.js

```diff
@@ -2,27 +2,32 @@
 
 exports._getNetworkId = conn => () => conn.getNetworkId();
 
-exports._getUtxos = maybe => conn => () =>
-  conn.getUtxos().then(res => (res === null ? maybe.nothing : maybe.just(res)));
+exports._getUtxos = maybe => conn => amount => paginate => () =>
+  conn
+    .getUtxos(amount, paginate)
+    .then(res => (res === null ? maybe.nothing : maybe.just(res)))
+    .catch(_ => maybe.nothing);
```

Comment:
I do not understand what you mean. catches below use throw, which is not applicable here, as far as I understand.

Reply:
I'm referring to when we `catch` the errors and return `Nothing` without logging what was caught. If we could use our existing logging functionality to log the errors (or not, depending on the Contract config) instead of silencing them completely, I think that would be better for debugging

Reply:
We don't care about errors at this stage